mappings/net/minecraft/network/packet/c2s/play/PlayerAndEntityInteractedC2SPacket.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_ckudzhjz net/minecraft/network/packet/c2s/play/PlayerInteractEntityC2SPacket
+CLASS net/minecraft/unmapped/C_ckudzhjz net/minecraft/network/packet/c2s/play/PlayerAndEntityInteractedC2SPacket
```

Comment:
Not a huge fan of this change, maybe `PlayerInteractedWithEntity`, and the `WithX` being a modifier that can be added to the end of packet names in rare cases.

Reply:
I like this idea, will add it!